src/test/hertzbeat/templates/manager/configmap.yaml

```diff
@@ -0,0 +1,246 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ include "hertzbeat.manager" . }}"
+  labels:
+    {{- include "hertzbeat.labels" . | nindent 4 }}
+data:
+  application.yml: |+
+    server:
+      port: 1157
+    spring:
+      application:
+        name: ${HOSTNAME:@hertzbeat@}${PID}
+      profiles:
+        active: prod
+      mvc:
+        static-path-pattern: /**
+      jackson:
+        default-property-inclusion: ALWAYS
+      web:
+        resources:
+          static-locations:
+            - classpath:/dist/
+            - classpath:../dist/
+      # need to disable spring boot mongodb auto config, or default mongodb connection tried and failed..
+      autoconfigure:
+        exclude: org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration, org.springframework.boot.autoconfigure.data.mongo.MongoDataAutoConfiguration
+      thymeleaf:
+        prefix: classpath:/templates/
+        check-template-location: true
+        cache: true
+        suffix: .html
+        mode: HTML
+        template-resolver-order: 1
+    
+    management:
+      health:
+        mail:
+          enabled: off
+      endpoints:
+        web:
+          exposure:
+            include:
+              - 'metrics'
+              - 'health'
+              - 'env'
+        enabled-by-default: on
+    
+    sureness:
+      auths:
+        - digest
+        - basic
+        - jwt
+      jwt:
+        secret: 'CyaFv0bwq2Eik0jdrKUtsA6bx3sDJeFV643R
```

Comment:
Hard code the secret key is not a good chooice.

Reply:
ok 🫡 I will fix it.